ethportal-peertest/src/scenarios/offer_accept.rs

```diff
@@ -69,7 +69,6 @@ pub fn test_offer_accept(peertest_config: PeertestConfig, peertest: &Peertest) {
     let received_content_str = received_content_value.as_str().unwrap();
     assert_eq!(
         HISTORY_CONTENT_VALUE, received_content_str,
-        "The received content {}, must match the expected {}",
-        HISTORY_CONTENT_VALUE, received_content_str,
+        "The received content {HISTORY_CONTENT_VALUE}, must match the expected {received_content_str}",
```

Comment:
These two values should actually be switched around

Reply:
I think you're right. This was a nightly-clippy change so after reverting I will leave it untouched for now.